### PR TITLE
feat: Added documentation for integrating Flower with Authelia

### PR DESCRIPTION
**Pull request by m4rc3l-h3**
This PR adds a description of how to integrate a Flower fork with Authelia as authorization server. As flower does not seem to be actively maintained any longer (see [discussion 1368](https://github.com/mher/flower/discussions/1368), a reference to the fork was added.

### CODE REVIEW

File: docs/content/integration/openid-connect/flower/index.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+This example makes the following assumptions:
+
+* __Application Root URL:__ `https://flower.{{< sitevar name="domain" nojs="example.com" >}}/`
+* __Authelia Root URL:__ `https://{{< sitevar name="subdomain-authelia" nojs="auth" >}}.{{< sitevar name="domain" nojs="example.com" >}}/`
+* __Client ID:__ `flower`
+* __Client Secret:__ `insecure_secret`
+* __Authentication Name (Flower):__ `authelia`:
+    * This option determines the redirect URI in the format of
+      `https://flower.{{< sitevar name="domain" nojs="example.com" >}}/login`.
+      This means if you change this value you need to update the redirect URI.
```
Comment (james-d-elliott) [inline docs/content/integration/openid-connect/flower/index.md:41]: Not sure where the `authelia` element fits in here?

**Review by james-d-elliott — COMMENTED** [review]
LGTM just a few questions. In above to the individual comments how does one configure the Authelia URLs for Flower, there seems to be no reference to the issuer etc at all.

File: docs/content/integration/openid-connect/flower/index.md (hunk 1 above)
Comment (m4rc3l-h3) [reply docs/content/integration/openid-connect/flower/index.md]: Hi @james-d-elliott 
Thank you for your comments and sorry for the inconveniences. I have updated both aspects by a.) reworking the assumptions section regarding what is required from Flower and b.) added another section in the Application configuration to set the environment variable for flower.

**Review by james-d-elliott — APPROVED** [review]

**Review by james-d-elliott — APPROVED** [review]

File: docs/content/integration/openid-connect/flower/index.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        userinfo_signed_response_alg: 'none'
+        token_endpoint_auth_method: 'client_secret_post'
+```
+
+### Application
+
+
+To configure [Flower] to utilize Authelia as an [OpenID Connect 1.0] Provider you have to update the `flowerconfig.py` configuration file and configure the `FLOWER_OAUTH2_AUTHELIA_BASE_URL` environment variable.
+
+#### Configuration File
```
Comment (james-d-elliott) [inline docs/content/integration/openid-connect/flower/index.md:78]:
```suggestion
#### Configuration File

```

File: docs/content/integration/openid-connect/flower/index.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+Add the following values to [Flower] `flowerconfig.py`:
+```python
+auth = '.*@{{< sitevar name="domain" nojs="example.com" >}}'
+auth_provider = 'flower.views.auth.AutheliaLoginHandler'
+oauth2_key = 'flower'
+oauth2_secret = 'insecure_secret'
+oauth2_redirect_uri = 'https://flower.{{< sitevar name="domain" nojs="example.com" >}}/login'
+```
+
+#### Environment Variables
```
Comment (james-d-elliott) [inline docs/content/integration/openid-connect/flower/index.md:88]:
```suggestion
#### Environment Variables

```

**Pull request merged**